spec/e2e/features/short_form/error_messages.feature

```diff
@@ -80,3 +80,15 @@ Feature: Short Form Application
         # error: uploaded preference document too large
         When I upload a too-large "Gas bill" as my proof of preference for "liveInSf"
         Then I should see an error about the file being too large
+        And I opt out of "Live/Work" preference
+
+        # error: address not entered for Alice Griffith
+        When I select "Jane Doe" for "aliceGriffith" preference
+        And I hit the Next button "1" time
+        Then I should see a blank address error
+
+# error: address not found for Alice Griffith
```

Comment:
@creativecoder should we delete these?

Reply:
Whoops, meant to use those in #1041... will go update that PR now.